.github/workflows/code-check.yml

```diff
@@ -10,10 +10,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Setup Biome
-        uses: biomejs/setup-biome@v2
       - name: Run Biome
-        run: biome ci --reporter=github .
+        run: npx biome ci --reporter=github .
```

Comment:
This doesn't seem to feed it's output to github properly, behind the scene that action does more than running `ci`. What went wrong that requires this change?

Reply:
We don't know :D
The GH action couldn't run for some reason, but we couldn't see any error messages

Reply:
Interesting. It looks like the `ci` exits the moment error happens. Is there a local error, that could be triggering the      GH action?

Reply:
`npx biome ci --diagnostic-level=error` gave out a small formatting error, checking if that is the cause

Reply:
Approved your PR, ty! Would be good to see this kind of errors on GitHub somehow tho :D

Reply:
Now it shows errors on my PR, but they are invalid? Something is fishy with this action. :fish:

Reply:
😵‍💫